cli_tools/test_data/test_import_ovf.wf.json

```diff
@@ -0,0 +1,106 @@
+{
+  "Name": "import-ova",
+  "DefaultTimeout": "240m",
+  "Vars": {
+    "boot_disk_file": {
+      "Required": true,
+      "Description": "The GCS path to the virtual disk to import."
+    },
+    "instance_name": {
+      "Required": true,
+      "Description": "The name of the imported instance."
+    },
+    "translate_workflow": {
+      "Required": true,
+      "Description": "The path to the translation workflow to run."
+    },
+    "install_gce_packages": {
+      "Value": "true",
+      "Description": "Whether to install GCE packages."
+    },
+    "description": {
+      "Value": "",
+      "Description": "Optional description to set for the instance."
+    },
+    "translation_disk_name": "temp-translation-disk-${ID}",
+    "boot_image_name": "boot-image-${ID}",
+    "boot_image_name_untranslated": "boot-image-untranslated-${ID}",
+    "machine_type": "n1-standard-1",
+    "network": {
+      "Value": "global/networks/default",
+      "Description": "Network to use for the import instance"
+    },
+    "subnet": {
+      "Value": "",
+      "Description": "SubNetwork to use for the import instance"
+    },
+    "private_network_ip": {
+      "Value": "",
+      "Description": "IP to assign to the instance"
+    },
+    "network_tier": {
+      "Value": "PREMIUM",
+      "Description": "Network tier that will be used to configure the instance"
+    }
+  },
+  "Steps": {
+    "create-boot-disk": {
+      "CreateDisks": [
+        {
+          "Name": "${instance_name}-boot-disk",
+          "SourceImage": "${boot_image_name}",
+          "Type": "pd-ssd",
+          "ExactName": true,
+          "NoCleanup": true
+        }
+      ]
+    },
+    "create-image": {
+      "CreateImages": [
+        {
+          "Name": "${image_name}",
+          "SourceDisk": "${import_disk_name}",
+          "Family": "${family}",
+          "Description": "${description}",
+          "ExactName": true,
+          "NoCleanup": true
+        }
+      ]
+    },
+    "create-instance": {
+      "CreateInstances": [
+        {
+          "Name": "${instance_name}",
+          "Disks": [
+            {
+              "Source": "${instance_name}-boot-disk",
+              "AutoDelete": true,
+              "boot": true
+            }
+          ],
+          "MachineType": "${machine_type}",
+          "networkInterfaces": [
+            {
+              "network": "${network}",
```

Comment:
What is the purpose of network/subnetwork/networkip for this testing?

Reply:
In a single project there could be multiple networks, with various configurations. Users should be able to choose which network newly created instance should use.